packages/lib/services/interop/InteropService.ts

```diff
@@ -136,13 +132,15 @@ export default class InteropService {
 					target: FileSystemItem.File,
 					isNoteArchive: false,
 					description: _('HTML File'),
-				}, () => new InteropService_Exporter_Html()),
+					supportsMobile: false,
+				}, dynamicRequireModuleFactory('InteropService_Exporter_Html')),
 
 				makeExportModule({
 					format: 'html',
 					target: FileSystemItem.Directory,
 					description: _('HTML Directory'),
-				}, () => new InteropService_Exporter_Html()),
+					supportsMobile: false,
+				}, dynamicRequireModuleFactory('InteropService_Exporter_Html')),
```

Comment:
Many of the importers/exporters use `fs` directly and not through `shim`. As such, rather than trying to fix these issues now, I've reverted the broken modules to use dynamic requires.

(See `TODO` comment in `Module.ts`).